### PR TITLE
🗄 Create Server README.md

### PR DESCRIPTION
**Pull request by jeffzh4ng**
## Purpose
Creates README for server and LICENSE.md for entire monorepo.

### CODE REVIEW

File: server/src/index.ts [hunk 1]
```diff
@@ -31,7 +31,7 @@ const bootstrap = async () => {
 
   app.use((_req, res) => res.status(404).json({ message: 'No route found' }))
   app.listen(port, () => {
-    console.log(`Waterpark REST API server running on port ${port} 🦆`)
+    console.log(`Waterpark REST API server running on  http://localhost:${port}/api/v1 🦆`)
```
Comment (xujustinj) [inline server/src/index.ts:34]:
```suggestion
    console.log(`Waterpark REST API server running on http://localhost:${port}/api/v1 🦆`)
```

**Review by lesliexin — APPROVED** [review]
🦆

File: server/README.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+DB_PASSWORD=loolabs
+DB_PORT=5432
+DATABASE_URL=postgresql://loolabs:loolabs@db/clubs
+```
+
+### Installing
+
+1. Clone the mono repo.
+1. Navigate to the server directory with `cd server`.
+1. Run `docker-compose up` to start the express and postgres servers.
```
[the remark this answers is not in the document]
Comment (jeffzh4ng) [reply server/README.md:27]: You're right, great catch.

File: server/README.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+You should see messages in your terminal from the `server` and `postgres` containers, and finally be greeted with:
+
+![](https://i.imgur.com/WhvhzFs.png)
```
[the remark this answers is not in the document]
Comment (jeffzh4ng) [reply server/README.md]: Smart man, TIL you can have local images in markdown. I moved all images to our repo.

File: server/README.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+DB_PASSWORD=loolabs
+DB_PORT=5432
+DATABASE_URL=postgresql://loolabs:loolabs@db/clubs
+```
+
+### Installing
+
+1. Clone the mono repo.
+1. Navigate to the server directory with `cd server`.
+1. Run `docker-compose up` to start the express and postgres servers.
```
Comment (xujustinj) [inline server/README.md:27]:
```suggestion
1. Clone the `waterpark` repository.
1. In the root directory of `waterpark`, run `docker-compose up` to start the express and postgres servers.
```

**Review by KTong821 — DISMISSED** [review]
lgtm

File: server/README.md (hunk 4 above)
Comment (jeffzh4ng) [reply server/README.md:27]: Woops, thanks.

**Review by xujustinj — APPROVED** [review]

**Review by KTong821 — APPROVED** [review]

**Pull request merged**